TheraLang.Web/Controllers/AccountController.cs

```diff
@@ -14,35 +14,39 @@ public class AccountController : ControllerBase
     {
 
         private readonly IAuthenticateService _authService;
-        public AccountController(IAuthenticateService authService)
+        private readonly IUserManagementService _userManagementService;
+        private readonly IDataSeedService _dataSeedService;
+        public AccountController(IAuthenticateService authService, IUserManagementService userManagementService, IDataSeedService dataSeedService)
         {
             _authService = authService;
+            _userManagementService = userManagementService;
+            _dataSeedService = dataSeedService;
```

Comment:
Shouldn't we use data seeding in separate project that we already have? Cause seed data in account controller is not obvious at all.